extensions/vertx-http/deployment/pom.xml

```diff
@@ -86,6 +102,89 @@
                     </annotationProcessorPaths>
                 </configuration>
             </plugin>
+            
+            <!-- So we don't have to keep a version of these javascript libs in our repo -->
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <version>${maven-dependency-plugin.version}</version>
```

Comment:
I would have expected the version to be set in one of the parents so that you don't have to define it here.

I see it's already defined in JBoss Parent so my guess is that you can just drop the version from here.

Reply:
Yes, sorry. done :)